packages/shared/src/modals/ConnectWalletDialog/ConnectWallet.tsx

```diff
@@ -69,22 +64,20 @@ export function ConnectWalletDialog() {
             }
         }
 
-        setConnectWalletDialog({
-            open: false,
-        })
+        ConnectWalletDialog.open()
```

Comment:
Should be `close`, isn't it?
```suggestion
        ConnectWalletDialog.close()
```